### PR TITLE
Add lint format hook

### PR DESCRIPTION
**Pull request by jrwbabylonchain**

### CODE REVIEW

**Review by jeremy-babylonchain — APPROVED** [review]
looks good to me!

File: src/app/api/getFinalityProviders.ts [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import { encode } from "url-safe-base64";
-import { apiWrapper } from "./apiWrapper";
+
 import { FinalityProvider } from "../types/finalityProviders";
 import { Pagination } from "../types/api";
```
Comment (gbarkhatov) [inline src/app/api/getFinalityProviders.ts:5]: Should we configure it differently? It adds an empty line between the local imports

File: src/app/components/Staking/Staking.tsx [hunk 2]
```diff
@@ -6,23 +6,25 @@ import { FinalityProvider as FinalityProviderInterface } from "@/app/types/final
...
 import { getStakingTerm } from "@/utils/getStakingTerm";
-import { FinalityProviders } from "./FinalityProviders/FinalityProviders";
 import { WalletProvider } from "@/utils/wallet/wallet_provider";
 import { isStakingSignReady } from "@/utils/isStakingSignReady";
 import { GlobalParamsVersion } from "@/app/types/globalParams";
 import { Delegation } from "@/app/types/delegations";
 import { LoadingView } from "@/app/components/Loading/Loading";
+import { useError } from "@/app/context/Error/ErrorContext";
+import { ErrorState } from "@/app/types/errors";
+
```
Comment (gbarkhatov) [inline src/app/components/Staking/Staking.tsx:16]: Or does it treat
1. Global package imports
2. Typescript path "@"
3. Local imports above "../"
4. Local imports "./"

**Review by gbarkhatov — APPROVED** [review]

File: src/app/components/Staking/Staking.tsx (hunk 2 above)
Comment (jrwbabylonchain) [reply src/app/components/Staking/Staking.tsx:17]: i think so.
I have added the import order in the prettierrc.json, we shall have consistent import ordering now

**Pull request merged**